docs/warnings/invalid-element-type.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Invalid Element Type Warning
+layout: single
+permalink: warnings/invalid-element-type.html
+---
+
+You probably came here because your code is trying to create a ReactElement with an invalid type using JSX or the `React.createElement` API. This usually happens when you have an invalid import statement.
+
+`React.createElement` requires an argument of type `string` (e.g. 'div', 'span'), or a `ReactClass`/`React.Component`. It cannot be of type `number`, `boolean`, `undefined` or `null`. See the documentation of this API: [https://facebook.github.io/react/docs/top-level-api.html](https://facebook.github.io/react/docs/top-level-api.html)
```

Comment:
Which API are you referring to? It would make sense to link to createElement() specifically. You can hover over its header in docs to see the exact anchor link.

Please also remove domain from the URL and just start it with /react. This is useful in case docs move.